Tests/Tests/RouterSpec.swift

```diff
@@ -450,6 +594,87 @@ class RouterSpec: QuickSpec {
                         expect(displayedTooltip.frame.origin).toEventually(equal(lastTooltipPosition.applying(translation)))
                     }
 
+                    it("will show tooltip's position in the correct position") {
```

Comment:
This test is unnecessary. UI behaviour should be tested in UI tests but in case of tooltip positioning it's just something to be tested in QA tests. Testing position is difficult and brings a big chunk of SDK logic copied into test spec. No unit test should rely on UI Constants defined in SDK classes.
Please remove this test OR just check if `displayedTooltip` frame is different for each position - that's enough.

Reply:
Because we need to cover each position in `updateFrame` function, I changed the case to check whether the tooltip is displayed in a different position.